ci/Dockerfile.dev

```diff
@@ -2,19 +2,19 @@ FROM continuumio/miniconda3
 
 RUN apt-get -qq update --yes \
  && apt-get -qq install --yes --no-install-recommends \
-    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre \
+    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-11-jre \
  && rm -rf /var/lib/apt/lists/*
 
 ARG PYTHON
 ADD ci/requirements-$PYTHON-dev.yml /
 
-RUN conda config --add channels conda-forge \
-  && conda update --all --yes --quiet \
-  && conda env create --name ibis-env --file /requirements-$PYTHON-dev.yml \
-  && conda install --yes conda-build \
-  && conda clean --all --yes
+RUN /opt/conda/bin/conda config --add channels conda-forge \
+  && /opt/conda/bin/conda update --all --yes --quiet \
+  && /opt/conda/bin/conda env create --name ibis-env --file /requirements-$PYTHON-dev.yml \
+  && /opt/conda/bin/conda install --yes conda-build \
```

Comment:
and maybe you can add something like:

`  && /opt/conda/bin/conda install -n ibis-env --yes openjdk=8 \`

Reply:
I placed the specification of the JDK in dev requirements files.